packages/lumx-react/src/components/web-bookmark/WebBookmark.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import classNames from 'classnames';
+
+import { AspectRatio, Theme, Thumbnail } from '@lumx/react';
+
+import { COMPONENT_PREFIX } from '@lumx/react/constants';
+import { IGenericProps, getRootClassName, handleBasicClasses } from '@lumx/react/utils';
+import { Alignment, Size } from '..';
+
+/////////////////////////////
+
+/**
+ * Defines the props of the component.
+ */
+interface IWebBookmarkProps extends IGenericProps {
+    /* The url of the link. */
+    url: string;
+    /* Content text. Can be either a string, or sanitized html. */
+    description?:
+        | string
+        | {
+              __html: string;
+          };
+    /* The size variant of the web bookmark. */
+    size?: Size.regular | Size.big;
+    /* Theme. */
+    theme?: Theme;
+    /* Thumbnail image source */
+    thumbnail?: string;
```

Comment:
Does the component displays correctly without a thumbnail? Should it be required here?

Reply:
Yes it does, it just leaves an empty space. But maybe we should add a placeholder. @FlorentDinet ?